mocket/utils.py

```diff
@@ -38,23 +47,26 @@ def hexload(string):
     return encode_to_bytes(binascii.unhexlify(string_no_spaces))
 
 
-def get_mocketize(wrapper_):
+def get_mocketize(wrapper_: Callable) -> Callable:
     import decorator
 
-    if decorator.__version__ < "5":  # pragma: no cover
+    if decorator.__version__ < "5":  # type: ignore[attr-defined] # pragma: no cover
         return decorator.decorator(wrapper_)
-    return decorator.decorator(wrapper_, kwsyntax=True)
+    return decorator.decorator(  # type: ignore[call-arg] # kwsyntax
+        wrapper_,
+        kwsyntax=True,
+    )
 
 
 class MocketMode:
-    __shared_state = {}
-    STRICT = None
-    STRICT_ALLOWED = None
+    __shared_state: ClassVar[dict[str, Any]] = {}
+    STRICT: ClassVar = None
+    STRICT_ALLOWED: ClassVar = None
 
-    def __init__(self):
+    def __init__(self) -> None:
         self.__dict__ = self.__shared_state
 
-    def is_allowed(self, location: Union[str, Tuple[str, int]]) -> bool:
+    def is_allowed(self, location: str | tuple[str, int]) -> bool:
```

Comment:
Not sure why this was changed.

Reply:
I'm not changing the type I'm just using the new union syntax.

https://peps.python.org/pep-0604/

I can revert this if desired, but the type hasn't changed.